doc/contribute/contributor_expectations.rst

```diff
@@ -11,14 +11,46 @@ benefits:
   to set aside a few minutes to review smaller changes several times than it is
   to allocate large blocks of time to review a large PR.
 
-- Less wasted work if reviewers or maintainers reject the direction of the
-  change.
+- Contentious and long discussions have a more restricted "splash damage"
+  because they cannot drown unrelated topics that have successfully escaped
+  in separate Pull Requests.
 
 - Easier to rebase and merge. Smaller PRs are less likely to conflict with other
   changes in the tree.
 
+- Mixing different authors in the same pull request adds :ref:`extra
+  complications <modifying_contributions>`.
+
+- When independent of each other, smaller PRs can progress *concurrently*.
+  Disagreements, nitpicks and other delays in one place do not hold back
+  everything else.
+
+- Even when smaller PRs are dependent of each other, you can start reviews
+  and getting code merged before the whole work is complete. This faster
+  approach is known as "Stacked Diffs".
+
+- Less wasted work if reviewers or maintainers reject the direction of the
```

Comment:
how is this considered a "benefit"? If the direction is rejected, and part of it were already merged, that is not considered a good thing

Reply:
Yeah, I would remove it honestly.